package.json

```diff
@@ -90,7 +90,7 @@
     "flexboxgrid": "^6.3.0",
     "immutable": "^3.8.1",
     "lodash": "^4.5.1",
-    "lzma-native": "^1.5.2",
+    "lzma-native": "^2.0.2",
```

Comment:
Oh right, that one – well, we should still be safe & sound as that made it into the node / npm dists.